### PR TITLE
[CI] Build Docker Image and Github Action for CORD Demo Script

### PR DESCRIPTION
**Pull request by roynrishingha**
- The docker image is built using `buildx`.
- A secret variable named `DOCKER_PASSWORD` must be present in the repository settings.

### CODE REVIEW

**Comment by amarts** [comment]
I added a organization level variable called `DOCKER_TOKEN` can you use that instead?

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ env.USERNAME }}
+          password: "${{ secrets.DOCKER_PASSWORD }}"
```
Comment (amarts) [inline .github/workflows/release.yml:32]: ok, i will add DOCKER_PASSWORD itself and see merging this. Looks good otherwise.

**Comment by roynrishingha** [comment]
@amarts and @NiranjanAP I have tested this workflow on my fork and tested the Docker image as well. It is designed with `buildx` therefore will run on multiple architecture.

To test it locally try this:
```sh
docker run dhiwayroy/cord-demo-scripts demo
```

Previously @Rithwik0480 was working to build the Dockerfile and Github Action to deploy the image. But I found his approach to be not correct. So I wrote the `Dockerfile` and `Github Action` from scratch.
This PR both adds the `Dockerfile` and release using `Github Action`.

**Review by amarts — APPROVED** [review]

**Comment by amarts** [comment]
@adi-a11y  @Rithwik0480 @roynrishingha `docker run dhiway/cord-demo-scripts demo` is working now. Good job.

can we fix https://github.com/dhiway/sahaaya/pull/39 to mention only above command? I don't want someone to `build` images. And all `<image-name>` in that document should be replaced with `dhiway/cord-demo-scripts`.

File: .github/workflows/release.yml (hunk 1 above)
Comment (roynrishingha) [reply .github/workflows/release.yml:32]: If `DOCKER_TOKEN` is the password to login to docker hub using dhiway as a user, then it can be replaced.

**Pull request merged**